contracts/native/version-control/src/queries.rs

```diff
@@ -1017,77 +1003,25 @@ mod test {
         }
     }
 
-    mod list_namespaces {
+    mod query_namespaces {
         use super::*;
 
-        fn filtered_list_msg(filter: NamespaceFilter) -> QueryMsg {
-            QueryMsg::NamespaceList {
-                filter: Some(filter),
-                start_after: None,
-                limit: None,
-            }
-        }
-
         #[test]
-        fn filter_namespaces() {
+        fn namespaces() {
             let mut deps = mock_dependencies();
             deps.querier = mock_manager_querier().build();
             init_with_mods(deps.as_mut());
 
-            // add namespaces as others
-            let namespaces = vec![
-                "other1".to_string(),
-                "other2".to_string(),
-                "other3".to_string(),
-            ];
-            add_namespaces(
-                deps.as_mut(),
-                namespaces.clone(),
-                TEST_OTHER_ACCOUNT_ID,
-                TEST_OTHER,
+            // get for test other account
+            let res = query_helper(
+                deps.as_ref(),
+                QueryMsg::Namespaces {
```

Comment:
Should we change that enum member to query a single namespace ? That makes more sense I think

Reply:
Well you still could want to query multiple account id's and their namespaces, no?

Reply:
yes ok